src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -235,18 +235,45 @@ const myFade = {
 <header transition:animate={myFade}> ... </header>
 ```
 
-## Preventing client-side navigation
+## Router control
 
-Once you have enabled client-side routing on a page by adding the `<ViewTransitions />` component, every anchor on the page that links to another page on your site will be navigated via client-side routing. There are some cases where you might want to not navigate via CSR. One example is if the link is to any non-page, such as a PDF within your `public/` folder. Or an API route that produces an image.
+The `<ViewTransitions />` router handles navigation by listening to:
+
+- Clicks on `<a>` elements.
+- Backwards and forwards navigation events.
+
+The following sections discuss how you can further control when navigation occurs within the router.
+
+### Preventing client-side navigation
+
+There are some cases where you might want to not navigate via CSR. A transition will only occur when both the old and new page has the `<ViewTransitions />` component. There may be cases where you want a nice animation coming from one page, such as going from an article listing page `/articles` to an article detail page `/articles/emperor-penguins`, but you don't want an animation when coming from somewhere else on the site.
 
 In these cases, you can opt out of client-side routing on a per-link basis using the `data-astro-reload` attribute like so:
```

Comment:
I agree that the need to have `<ViewTransitions>` at both ends is part of the top of the page.

Instead of 
> Astro provides a `<ViewTransitions>` routing component that can be added to the `<head>` of a single page to control page transitions when you navigate to another page. 

we could say:
> Astro provides a `<ViewTransitions />` routing component that can be added to the `<head>` of any page. The view will transition seamlessly when you navigate between two pages that both contain the <ViewTransitions /> component.

Reply:
Yeah, this is an improvement, can move this piece of information as part of a refactor.